partitioner.go

```diff
@@ -82,18 +82,20 @@ func (p *roundRobinPartitioner) RequiresConsistency() bool {
 	return false
 }
 
+type hasherFunc func() hash.Hash32
+
 type hashPartitioner struct {
 	random Partitioner
 	hasher hash.Hash32
 }
 
 // NewCustomHashPartitioner is a wrapper around NewHashPartitioner,
 // allowing the use of custom hasher
```

Comment:
Absolutely. I might have gotten a bit in a hurry to open the PR, since I realised my mistake.